lib/trace-agent.js

```diff
@@ -254,15 +254,17 @@ TraceAgent.prototype.parseContextFromHeader = function(str) {
  * @param {SpanData} spanData The span to be added to headers.
  * @param {Object} headers The http headers associated with the
  *   current request.
+ * @param {boolean} traced Whether this request was traced by the agent.
  */
-TraceAgent.prototype.addContextToHeaders = function(spanData, headers) {
+TraceAgent.prototype.addContextToHeaders = function(spanData, headers, traced) {
   if (spanData === SpanData.nullSpan) {
     return;
   }
   var header = spanData.trace.traceId + '/' + spanData.span.spanId;
-  if (spanData.options) {
-    header += (';o=' + spanData.options);
-  }
+  var options =
+    traced ? spanData.options | constants.TRACE_OPTIONS_TRACE_ENABLED :
```

Comment:
[Style Guide](https://google.github.io/styleguide/javascriptguide.xml?showone=Code_formatting#Code_formatting): Put the `traced ?` on the preceeding line to make the two legs at similar indent.